### ISSUE TITLE
False positive on condition-wrapping

### ISSUE THREAD

**Issue by eygraber** [pinterest/ktlint#2446]
## Expected Behavior
The following code should not trigger a `condition-wrapping` violation:

```kotlin
    !isLoggingOut && currentUser.isIdValid() && loadTokenAndDeviceId().let { (token, deviceId) ->
      token != null && token != deviceId
    }
```

## Observed Behavior
It does trigger a violation

## Your Environment
* Version of ktlint used: 1.1.0
* Operating System and version: Ubuntu 22.04

```
[*.{kt,kts}]
ij_kotlin_allow_trailing_comma = true
ij_kotlin_allow_trailing_comma_on_call_site = true
ij_kotlin_imports_layout=*,java.**,javax.**,kotlin.**,^
indent_size = 2
indent_style = space
insert_final_newline = true
ktlint_code_style = android_studio
ktlint_experimental = enabled
ktlint_ignore_back_ticked_identifier = true
ktlint_standard_annotation = disabled
ktlint_standard_class-signature = disabled
ktlint_standard_comment-wrapping = disabled
ktlint_standard_discouraged-comment-location = disabled
ktlint_standard_filename = disabled
ktlint_standard_function-literal = disabled
ktlint_standard_function-naming = disabled
ktlint_standard_function-signature = disabled
ktlint_standard_keyword-spacing = disabled
ktlint_standard_package-name = disabled
ktlint_standard_property-naming = disabled
ktlint_standard_spacing-between-declarations-with-annotations = disabled
ktlint_standard_trailing-comma-on-call-site = disabled
ktlint_standard_trailing-comma-on-declaration-site = disabled
max_line_length = 120
```

**Comment by paul-dingemans** [comment]
This is not a false positive but intended behavior.

Subexpression below is a multiline expression, which according to this rule should start on a separate line.
```
loadTokenAndDeviceId().let { (token, deviceId) ->
      token != null && token != deviceId
}
```

Your example nicely demonstrates a disconnect the first and second line of the subexpression above. It first character is positioned at column 49 while the last character of the second line is at position 39. Like with other multiline expression, ktilint prefers that each line in a subexpression has the same offset from start of the line.
